<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeeChatGUI</class>
 <widget class="QMainWindow" name="SpeeChatGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpeeChatGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="ChatDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MsgLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your message:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="MsgEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendBtn">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>280</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
